code/modules/hydroponics/hydroitemdefines.dm

```diff
@@ -230,11 +230,12 @@
 	hitsound = 'sound/items/weapons/bladeslice.ogg'
 
 ///Catch right clicks so we can stylize!
-/obj/item/secateurs/pre_attack_secondary(atom/target, mob/living/user, params)
+/obj/item/secateurs/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
```

Comment:
You want `interact_with_atom_secondary` unless you're intentionally moving it to lmb

Reply:
This was intentional. I kept stabbing myself in testing and figured it was probably easier for everyone if I just put it on lmb and rmb

Reply:
Ok, but this doesn't break using secateurs on actual plants, right?

You catch-all block further interactions which might

Reply:
I’ll have to check, will be a bit

Reply:
Looks like you were right. I've put secateurs back on right click, flesh reshaper and flesh reshaper on left and right click